### ISSUE TITLE
Max-Logits evaluation way

### ISSUE THREAD

**Issue by ghost** [robin-chan/meta-ood#2]
Hi, 

Thanks for your work. Based on your code, I'm trying to distinguish the FPR95 based on the Max-Logits instead of the entropy map. Could you please give us some advice for the modification of the code. 

I'm totally new for the OoD Segmentation.

Cheers

**Comment by robin-chan** [comment]
Hello,

if you really want to use my code, you need to modify the prediction function here: https://github.com/robin-chan/meta-ood/blob/8657a5e748803998351caa23d44bb95a40b6e32f/src/model_utils.py#L39 to return logits instead of softmax. Note that, however, my implementation of the FPR95 requires normalized OoD scores, ie scores in [0,1].

It would be the easiest to use the following benchmark code: [SegmentMeIfYouCan/road-anomaly-benchmark: Benchmark of detection methods for anomalies and obstacles in traffic images. (github.com)](https://github.com/SegmentMeIfYouCan/road-anomaly-benchmark)

In this script [road-anomaly-benchmark/some_methods_inference_public.py at master · SegmentMeIfYouCan/road-anomaly-benchmark (github.com)](https://github.com/SegmentMeIfYouCan/road-anomaly-benchmark/blob/master/some_methods_inference_public.py) the pretrained weights are also already included and you just need to modify https://github.com/SegmentMeIfYouCan/road-anomaly-benchmark/blob/2ffd4ec0026196c363722535da1638d3a64bb514/methods/baselines.py#L93 to return the max logits instead of the entropy.


Best,
Robin
